src/cryptonote_core/service_node_list.cpp

```diff
@@ -533,17 +602,26 @@ namespace service_nodes
       add_to_winners(m_service_nodes_infos.at(key).addresses[i], m_service_nodes_infos.at(key).portions[i]);
       remaining_portions -= m_service_nodes_infos.at(key).portions[i];
     }
-    uint64_t remaining_staking_requirement = m_service_nodes_infos.at(key).staking_requirement;
-    for (const auto& contribution : m_service_nodes_infos.at(key).contributions)
-    {
-      uint64_t actual_contribution = std::min(remaining_staking_requirement, contribution.amount);
-      remaining_staking_requirement -= actual_contribution;
+    */
 
+    // sort contributors in order of the "order" field
+    const service_node_info& info = m_service_nodes_infos.at(key);
+    std::vector<cryptonote::account_public_address> addresses;
+    for (const auto& iter : info.contributors)
+      addresses.push_back(iter.first);
+    sort(addresses.begin(), addresses.end(),
```

Comment:
If this is `std::sort` it would be nice to explicitly prepend the `std::`

Reply:
I don't know how this compiles. changed.